src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/returncount/Example1.java

```diff
@@ -0,0 +1,43 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="ReturnCount">
+      <property name="max" value="3"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.returncount;
+
+// xdoc section -- start
+import java.util.function.Predicate;
+
+public class Example1 {
+    public Example1() {}
+
+    public Example1(int i) { return; }
+
+    public int signA(int x) {
+        if (x < -2) { return -1; }
+        return 0;
+    }
+
+    public int signB(int x) { // violation, 'max allowed for non-void methods/lambdas is 3'
```

Comment:
We need to cut message more or place comment above method and change wording to ` // violation below`, see examples in other tests.